code/datums/status_effects/buffs.dm

```diff
@@ -33,6 +36,8 @@
 	held_items += owner.l_hand
 	held_items += owner.r_hand
 	for(var/obj/item/his_grace/HG in held_items)
+		victims = HG.victims
+		victims_needed = HG.victims_needed
```

Comment:
I'm not a huge fan of mirroring values from one set of variables to another like this, it feels like there should only be one source of truth. It almost feels to me like it'd be nicer to include a reference to the toolbox in the status effect and just get the values right from it? It shouldn't be too hard to make something like this GC nicely. Regardless, it's probably worth another look.